.travis.yml

```diff
@@ -34,6 +34,19 @@ script:
   # Configure test script so we can run extra tests after playbook is run.
   - export container_id=$(date +%s)
   - export cleanup=false
+  # run ansible-lint
+  - pip install --user ansible
+  - ansible-galaxy install -r tests/requirements.yml
+  - pip install --user ansible-lint
+  - ln -s $TRAVIS_BUILD_DIR ~/.ansible/roles/role_under_test
+  - git clone https://github.com/ansible/galaxy-lint-rules.git
+  - mv galaxy-lint-rules ~/galaxy
+  - ansible-lint -r ~/galaxy/rules tests/test.yml
+    --exclude ~/.ansible/roles/Stouts.postfix
+    -x 404 -x 405
```

Comment:
what are `404` and `405`? Is there a way to have a general configuration for ignored files and warnings?

Reply:
yes, please use this: https://github.com/ansible/ansible-lint#configuration-file

Reply:
404 and 405 is errors about connection and i disabled it.

Reply:
404 Doesn't need a relative path in role

Reply:
i'll try using configuration file